src/controlplane/client/authenticator/authenticator.go

```diff
@@ -0,0 +1,154 @@
+package authenticator
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/newrelic/infra-integrations-sdk/log"
+	"github.com/newrelic/nri-kubernetes/v2/internal/config"
+	"github.com/newrelic/nri-kubernetes/v2/internal/discovery"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/transport"
+)
+
+const (
+	DefaultSecretNamespace = "default"
```

Comment:
this is not used anymore

Reply:
damm it we need linters to be a bit more strict !